Essentials/src/main/java/com/earth2me/essentials/User.java

```diff
@@ -732,6 +738,20 @@ public void checkActivity() {
         }
     }
 
+    private boolean isAfkAuto() {
+        if (afkAutoCache != null) {
+            return afkAutoCache;
+        }
+        return afkAutoCache = isAuthorized("essentials.afk.auto");
+    }
+
+    private boolean isKickExempt() {
+        if (kickExemptCache != null) {
```

Comment:
Should be `isAfkKickExempt` to avoid confusion with general kick exemption (same goes for fields)